### PR TITLE
Fix for #196

### PR DESCRIPTION
**Pull request by mkendall07**
Bidders will always get key value pairs that are set in the `default` (i.e. `hb_pb`, `hb_bidder`, `hb_adid`, `hb_size`) even if using `alwaysUseBid=true`. If they choice to use the same key as the default keys, the value will be overwritten.

### CODE REVIEW

**Comment by mkendall07** [comment]
@CarsonBanov 

Let me know what you think.

**Comment by CarsonBanov** [comment]
@mkendall07 Looks great!  I'll put some cosmetic comments in the diff, but this should work

File: src/bidmanager.js [hunk 1]
```diff
@@ -208,20 +208,8 @@ exports.getKeyValueTargetingPairs = function (bidderCode, custBidObj) {
...
-    custBidObj.alwaysUseBid = bidder_settings[bidderCode].alwaysUseBid;
-  }
-
-  //2) set keys from standard setting. NOTE: this API doesn't seeem to be in use by any Adapter currently
-  else if (defaultBidderSettingsMap[bidderCode]) {
-    setKeys(keyValues, defaultBidderSettingsMap[bidderCode], custBidObj);
-    custBidObj.alwaysUseBid = defaultBidderSettingsMap[bidderCode].alwaysUseBid;
-  }
-
   //3) set the keys from "standard" setting or from prebid defaults
```
Comment (CarsonBanov) [inline src/bidmanager.js]: Need to update these 3) 2) 1) comments at some point

File: test/spec/bidmanager_spec.js [hunk 2]
```diff
@@ -264,7 +264,7 @@ var bidmanager = require('../../src/bidmanager');
                         }
                 };
 
-                var expected = {"hb_bidder":  bidderCode, "hb_adid": adId,"hb_pb": 15.0 };
```
Comment (CarsonBanov) [inline test/spec/bidmanager_spec.js]: Need to change the (test description](https://github.com/prebid/Prebid.js/pull/295/files#diff-f9d389eae2d418abaaaabd456eee8a9cR213) since now it _should_ always inherit standard.

Comment (CarsonBanov) [inline test/spec/bidmanager_spec.js]: Maybe we could also add some tests to flesh out the edge cases that use `alwaysUseBid`.  I don't see any places where we are testing that behavior.

Comment (mkendall07) [inline test/spec/bidmanager_spec.js]: Yep, just wanted to get the PR in front of you. I will add some some test cases for `alwaysUseBid` now and clean up the comments before merging.

Comment (CarsonBanov) [inline test/spec/bidmanager_spec.js]: Great!  I assumed it was just a quick one so sorry for being pedantic before you've even had time to clean it up.  Thanks
